projects/netgrif-components-core/src/lib/navigation/navigation-double-drawer/abstract-navigation-double-drawer.ts

```
@@ -485,14 +475,14 @@ export abstract class AbstractNavigationDoubleDrawerComponent implements OnInit,
     }
 
     protected resolveCustomViewsInRightSide() {
-        if (!!this._childCustomViews[this._currentNode.path]) {
-            this.rightItems.push(...Object.values(this._childCustomViews[this._currentNode.path]));
+        if (!!this._childCustomViews[this.currentPath]) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)